packages/entities/entities-consumers/src/components/ConsumerList.vue

```diff
@@ -49,6 +49,31 @@
         </Teleport>
       </template>
 
+      <template
+        v-if="enableV2EmptyStates && config.app === 'konnect'"
```

Comment:
suggestion: can we move 'konnect' to a constant

Reply:
Looking at it, that kind of refactor would be too large and I can't see an easy way forward without making the change for all entities in a single PR. Can you create a separate ticket for this work?